opentelemetry-etw-logs/Cargo.toml

```diff
@@ -36,7 +36,13 @@ spec_unstable_logs_enabled = [
     "opentelemetry/spec_unstable_logs_enabled",
     "opentelemetry_sdk/spec_unstable_logs_enabled",
 ]
-default = []
+internal-logs = [
+    "tracing",
+    "opentelemetry/internal-logs",
+    "opentelemetry_sdk/internal-logs",
+]
+serde_json = ["dep:serde_json"]
```

Comment:
looks like we use "-" instead of "_" for feature flags everywhere. Lets stick with that for consistency?

Reply:
After checking more, "_" is used everywhere! "internal-logs" is the exception by using "-" !